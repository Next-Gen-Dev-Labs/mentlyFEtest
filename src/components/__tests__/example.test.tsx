import { describe, it, expect } from "vitest";
import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";

describe("Example Test", () => {
  it("should render hello text correctly", () => {
    render(<div>Hello Test</div>);
    expect(screen.getByText("Hello Test")).toBeInTheDocument();
  });

  it("should not find non-existent text", () => {
    render(<div>Hello Test</div>);
    expect(screen.queryByText("Not Here")).not.toBeInTheDocument();
  });

  it("should match snapshot", () => {
    const { container } = render(<div>Hello Test</div>);
    expect(container).toMatchSnapshot();
  });
});
