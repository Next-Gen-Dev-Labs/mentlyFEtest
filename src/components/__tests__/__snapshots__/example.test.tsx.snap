// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Example Test > should match snapshot 1`] = `
<div>
  <div>
    Hello Test
  </div>
</div>
`;
